python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -460,7 +460,15 @@ export default function EditorContainer({
           config: serverConfigRes.aiconfig,
         });
       } catch (err: unknown) {
-        const message = err instanceof Error ? err.message : null;
+        const message = (err as { message?: string }).message ?? null;
+
+        // TODO: Add ErrorOutput component to show error instead of notification
+        dispatch({
+          type: "RUN_PROMPT_ERROR",
+          id: promptId,
+          message: message ?? undefined,
```

Comment:
Nit: would you recommend including the `promptId` (which I think is prompt name?) in the error message to make it more clear to user?

Reply:
`promptId` is a ui-state-specific unique id for each prompt and not the same as the name -- I don't really think it would be helpful to show since we never expose the unique ids for the prompts in the UI. I can update to show the name (if it can be obtained)

Reply:
<img width="560" alt="Screenshot 2024-01-03 at 10 44 27 AM" src="https://github.com/lastmile-ai/aiconfig/assets/5060851/b04bef02-aad6-4361-bd6a-c951287f0232">